build.sbt

```diff
@@ -571,3 +570,12 @@ lazy val update2_12 = Seq(
     }
   }
 )
+
+lazy val xlint = Seq(
+  scalacOptions += {
+    CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2, 12)) => "-Xlint:-unused,_"
```

Comment:
@kailuowang @ceedubs perhaps we should keep an eye on [this PR](https://github.com/scala/scala/pull/5876) as it might help with Simulacrum-related warnings